frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -99,6 +100,10 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
   @override
   void initState() {
     super.initState();
+    final SettingsShortcutService settingsShortcutService =
+        SettingsShortcutService();
+    //load all saved shortcuts and update their commands when initializing the editor.
+    settingsShortcutService.loadShortcuts();
```

Comment:
Should this code be invoked only when opening the editor for the first time or when the keybinding has been changed?

Reply:
And this function is an asynchronous function. Is there no need to use await here(basically, we don't use await in `initState`)? 

1. consider changing this function to a static function?
2. consider moving this function to a more suitable location?

Reply:
This function is intended to load shortcuts when we open the editor for the first time, we want to load all the saved shortcuts from the file and update the corresponding standardShortcutEvents accordingly. I want to call it only once. I will see how I can refactor it.

Reply:
```
@override
void initState() {
  super.initState();
  initializeShortcuts();
}

Future<void> initializeShortcuts() async {
  final SettingsShortcutService settingsShortcutService =
      SettingsShortcutService();
  await settingsShortcutService.loadShortcuts();
}

```
@LucasXu0  Can I do this instead?